install_files/ossec-server/usr/share/doc/ossec-server/changelog.Debian

```diff
@@ -0,0 +1,5 @@
+ossec-server (0.2.1-1) unstable; urgency=low
```

Comment:
the whole `install_files/ossec-server` directory can be deleted. since it moved to a separate repo now